.github/workflows/raindrops-ci.yml

```diff
@@ -0,0 +1,56 @@
+name: Raindrops CI
+
+on:
+  pull_request:
```

Comment:
Cool thanks for explaining everything. Turbo repo pretty neat. Can we run some steps on every pr not just master?

Reply:
It runs every creating PR to master and every pushing to master.

Reply:
Ah gotcha I read that wrong my bad

Reply:
Once the build passes I'll stamp it